api/v1/model/src/commonMain/kotlin/JobStatus.kt

```diff
@@ -39,5 +39,8 @@ enum class JobStatus {
     FAILED,
 
     /** The job was processed successfully. */
-    FINISHED
+    FINISHED,
+
+    /** The job has finished but there were some severe issues. */
```

Comment:
IMO "severe" is relative here, esp. if the threshold becomes configurable later, and probably should be omitted.